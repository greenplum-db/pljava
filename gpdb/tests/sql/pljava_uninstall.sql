\! gppkg -q --all | sed -n '/^pljava-/s|.*|pljava|p'
-- start_ignore
\! gppkg -r pljava
-- end_ignore
\! gppkg -q --all | sed -n '/^pljava-/s|.*|pljava|p'
