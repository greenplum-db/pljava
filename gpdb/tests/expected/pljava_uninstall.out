\! gppkg -q --all | sed -n '/^pljava-/s|.*|pljava|p'
pljava
-- start_ignore
\! gppkg -r pljava
-- end_ignore
\! gppkg -q --all | sed -n '/^pljava-/s|.*|pljava|p'
